src/styles/index.scss

```diff
@@ -1,3 +1,81 @@
-body {
+body,
+html {
   margin: 0;
+  padding: 0;
+  display: flex;
+  align-items: center;
+  justify-content: center;
+  width: 100%;
+  height: 100%;
```

Comment:
Avoid setting styles directly to the `html` or `body` tags as it can cause conflicts if multiple components or libraries try to do the same. Consider using a class for global styles or a CSS reset file.